src/Helper/Wordpress.php

```diff
@@ -269,6 +279,21 @@ public static function getUTCDateTimeByTimestamp($timestamp) {
 		return $dto;
 	}
 
+	/**
+	 * This function does what probably the getUTCDateTimeByTimestamp was originally supposed to do.
+	 * @param $timestamp
+	 *
+	 * @return DateTime
+	 * @throws \Exception
+	 */
+	public static function convertTimestampToUTCDatetime($timestamp) {
```

Comment:
Dazu habe ich noch ein paar Änderungen gepusht (in einem eigenen Branch den du mergen kannst wenn du willst) @hansmorb